Lombiq.VueJs.Samples/Lombiq.VueJs.Samples.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App"/>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Vue.js module for Orchard Core - Samples</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2019, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Vue.js module for Orchard Core - Samples: Sample content that demonstrates the features of Lombiq Vue.js module for Orchard Core.</Description>
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <PackageTags>OrchardCore;Lombiq;AspNetCore;Vue.js</PackageTags>
+    <RepositoryUrl>https://github.com/Lombiq/Orchard-Vue.js</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/Lombiq/Orchard-Vue.js/Lombiq.VueJs.Samples/Readme.md</PackageProjectUrl>
+    <PackageLicenseFile>Licence.md</PackageLicenseFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.2.2"/>
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.2.2"/>
+    <PackageReference Include="OrchardCore.Module.Targets" Version="1.2.2"/>
+    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.2.2"/>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(NuGetBuild)' != 'true'">
+    <ProjectReference Include="..\Lombiq.VueJs\Lombiq.VueJs.csproj"/>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(NuGetBuild)' == 'true'">
+    <PackageReference Include="Lombiq.VueJS" Version="1.0.0"/>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Assets\Scripts\VueComponents\*.vue"/>

```

Comment:
Is this needed? Don't we have a props file for that?

Reply:
Not needed. I think I've added the props file later and forgot to update here.